### DISCUSSION TITLE
Custom color transitions

### DISCUSSION THREAD

**Discussion by chrizzzly87** [matteobruni/tsparticles#5334] [Ideas]
Hey there,

it would be neat to have an option to set color transitions during the particles lifetime.
This might be very helpful to simulate real-life effects (i.e. burning fire that might go from bright yellow - red - black) as it burns up.

It would also be nice to be able to change the hue/lightness/color based on the rotation/displacement to create more realistic lightening options (i.e. to simulate golden glitter or other things where the lightening changes depending on the angle/squishness of the particle rather than it's lifetime).

This would be great :)

**Comment by matteobruni** [comment]
This is something I already thought about, but never worked on it. Can you open an issue so I can track it easier? Thanks for the suggestion!
